api/src/main/java/io/opencensus/common/Duration.java

```diff
@@ -71,6 +71,15 @@ public static Duration fromMillis(long millis) {
   }
 
   /**
+   * Returns the milliseconds representation of this {@code Duration}.
+   *
+   * @return the milliseconds representation of this {@code Duration}.
+   */
+  public long toMillis() {
```

Comment:
It's mainly for computing ratio between two Durations. I've moved this method to implcore instead of the api package.